test/Microsoft.Azure.SignalR.Protocols.Tests/ServiceMessageEqualityComparer.cs

```diff
@@ -176,6 +180,30 @@ private static bool ClaimsEqual(Claim[] x, Claim[] y)
             return true;
         }
 
+        private static bool HeadersEqual(HeaderDictionary x, HeaderDictionary y)
+        {
+            if (x == null && y == null)
```

Comment:
check when `x` or `y` is `null`

Reply:
fixed.

Reply:
if `x == null && y == []` the code below will throw

Reply:
Fixed. `x?.Count != y?.Count` won't throw.